### DISCUSSION TITLE
Propriedade com erro "Acessibilidade inconsistente (Inconsistent accessibility)"

### DISCUSSION THREAD

**Discussion by rogerfsp** [balta-io/forum#410] [Tira Dúvidas] [answered]
Prezados,

Em Fundamentos da Orientação a Objetos /  Validando as regras de negócio (aos 02:30), ao criar a prop Subscriptions, o VSCODE sublinhou esta de vermelho. Não consegui entender o porquê do erro, que  em seu vídeo, não ocorre.
Por favor, me ajude. Parei neste ponto. Obrigado.

Erro apresentado:  Inconsistent accessibility: property type 'IList<Subscription>' is less accessible than property 'Student.Subscriptions'

using Balta.SharedContext;
namespace Balta.SubscriptionContext
{
    public class Student : Base
    {
        public string Name { get; set; }
        public string Email { get; set; }
        public User User { get; set; }

        public IList<Subscription> Subscriptions { get; set; }   // Nesta linha, Subscriptions fica sublinhado de vermelho.
    }
}

// A classe Subscription está assim:

using Balta.SharedContext;
namespace Balta.SubscriptionContext
{
    class Subscription : Base
    {
        public Plan Plan { get; set; }
    }
}

**Comment by malighieri** [comment]
Boa noite.
Pelo que estou vendo do código, sua classe Subscription está sem modificador de acesso. Isso faz com que ela assuma escopo **Internal**. Sendo assim, ao usar uma propriedade com modificador de acesso **Public** cujo tipo seja a classe Subscription, você tem uma inconsistência. Para corrigir basta colocar o modificador de acesso **Public** na sua classe **Subscription**, se este for mesmo o uso que quer fazer dela.

**Reply by rogerfsp** [reply, marked as answer]
Olá Miron, antes de tudo, agradeço muito a atenção. Você está certo, faltou o modificador de acesso. Perdão pela inexperiência. Agora funcionou. Muito obrigado!
